.github/workflows/rottenness.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow checks whether last release is not too old
+
+name: Check last release
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '59 23 * * 0'
+
+jobs:
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+  rottenness:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Check last release date
+      continue-on-error: true
+      run: |
+        latest_release="$(curl --silent https://api.github.com/repos/Samsung/CredSweeper/releases/latest)"
+        published_date=$(echo "${latest_release}" | jq --raw-output '.published_at')
+        release_age=$(( $(date +%s) - $(date --date="${published_date}" +%s) ))
+        if [ 0 -ge ${release_age} ]; then
+            echo "Probably, release: ${published_date} and current timezone were different"
+            release_age=0
+        fi
+        tag_name=$(echo "${latest_release}" | jq --raw-output '.tag_name')
+        if [ $(( 60 * 60 * 24 * 8 )) -gt ${release_age} ]; then
```

Comment:
It seems it checks 8 days, not a week.

BTW I think checking only a week is too short.. How about make duration to 2 weeks?

Reply:
Ok. 14 days applied